README.md

```diff
@@ -0,0 +1,47 @@
+
+# Introduction
+
+A Microsoft Windows & Mac OS program that makes your system Catalan language friendly by installing language packages and changing configuration settings.
+
+**Project:** [Catalanitzador de Softcatalà](https://www.softcatala.org/projectes/catalanitzador-de-softcatala/)
+
+---
+
+# Development
+
+## Windows Requirements
+
+- Microsoft Visual Studio 2010 Express Edition
+- Google Test 1.6 ([Google Test](https://code.google.com/p/googletest/))
+- Google Mocks 1.6 ([Google Mocks](https://code.google.com/p/googlemock/))
+
+Download and unzip Google Mocks 1.6 at `libs/gmock` and Google Test 1.6 at `libs/gtest`.
+
+Before building the libraries, follow these steps for all projects within the gmock solution:
+
+1. Go to project properties → **Configuration Properties** → **C++ Code Generation**
+2. Select **Multithreaded Debug Dll**
+
+For gtest, this should be the default configuration. If it isn't, repeat the same steps.
+
+To build gtest, use the `gtest-md.sln` solution.
+
+## Mac requirements
+
+- Code X 
+
+# Git configuration for contributing
+
+Git should be set up to use Unix line endings:
+
+```bash
+$ git config --global core.autocrlf input
+```
```

Comment:
You can add a .gitattibutes file with this configuration in the repo and will be applied automatically

Reply:
I did not know this, it is really cool. Done: https://github.com/Softcatala/Catalanitzador/pull/11/commits/f3dacb903415cdbde31b9e1b62d9792d5509717a